common.blocks/menu/menu.bemhtml

```diff
@@ -28,6 +28,8 @@ block('menu')(
                     }
                 };
 
+            if(!Array.isArray(ctx.content)) throw Error('menu: content must be an array');
```

Comment:
> `ctx.isArray`

:+1: 

> `Error` vs `new Error`

We don't use `new` with native constructors, like `Boolean`, `String`, etc